### DISCUSSION TITLE
Access to window.navigator and navigator.clipboard.readText()

### DISCUSSION THREAD

**Discussion by arhanechka** [webdriverio/webdriverio#9361] [Q&A]
The question is how to use it properly? await  browser.execute(`navigator.clipboard.readText()`) returns null

**Comment by christian-bromann** [comment]
`navigator.clipboard.readText()` returns a promise, so better to this:

```ts
await browser.executeAsync((done) => navigator.clipboard.readText().then(done, done))
```

**Comment by arhanechka** [comment]
Thank you. Seems the script works by this way, but I am getting javascript error: javascript error: Document is not focused.
